src/device.coffee

```diff
@@ -40,8 +40,25 @@ exports.getID = memoizePromise ->
 			throw new Error('Could not find this device?!')
 		return devices[0].id
 
+exports.shuttingDown = false
```

Comment:
They're different, as offline mode would still allow updates to be applied if they were set locally... this will change a lot with multicontainer anyways.